src/main/java/frc/robot/commands/AutoBalanceClimb.java

```diff
@@ -0,0 +1,40 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.ClimbingSubsystem;
+
+public class AutoBalanceClimb extends Command{
+    ClimbingSubsystem s_climbingSubsystem;
+    double leftSpeed;
+    double rightSpeed;
+
+    public AutoBalanceClimb(){
+        s_climbingSubsystem = ClimbingSubsystem.getInstance();
+        addRequirements(s_climbingSubsystem);
+    }
+
+    @Override
+    public void execute(){
```

Comment:
At some point, we might consider adding a counter to this to keep track of how long this command has been running.